README.md

```diff
@@ -53,6 +53,29 @@ metadata = drive.insert_file(metadata, upload_source: 'test.txt', content_type:
 # Download a file
 drive.get_file(metadata.id, download_dest: '/tmp/myfile.txt')
 ```
+
+An example to use the Content API (Google Merchant Center)
+
+```ruby
+require 'google/apis/content_v2'
+require 'googleauth' # https://github.com/googleapis/google-auth-library-ruby
+
+Content = Google::Apis::ContentV2 # Alias the module
+content = Content::ShoppingContentService.new
+
+scope = 'https://www.googleapis.com/auth/content'
+merchant_id = # Merchant ID found on dashboard
+
+content.authorization = Google::Auth::ServiceAccountCredentials.make_creds(
+  json_key_io: File.open('./content-api-key.json'),
+  scope: scope)
+
+content.authorization.fetch_access_token!
+# Service methods: https://github.com/googleapis/google-api-ruby-client/blob/master/generated/google/apis/content_v2/service.rb
```

Comment:
Can you link to the docs instead: https://googleapis.dev/ruby/google-api-client/latest/Google/Apis/ContentV2/ShoppingContentService.html

Reply:
Indeed! My fork is now updated.